pages/application.md

```diff
@@ -4,15 +4,15 @@ title: Application
 permalink: /application/
 ---
 
+To express interest in attending the workshop please fill out the
+[US-RSE Workshop Interest Form](https://forms.gle/E5BKkysrfQZ2EtXbA).
+The form will be open until at least December 21, 2021.
 
+Invitations are expected to be sent by mid-January.
 
+<br>
+
+### A note about the 2020 cancellation
 Thank you to everyone who expressed interest in attending the original
 workshop. Because of the time that has passed since the initial call
-for participation and the rescheduled workshop, we will reopen the
-call in late 2021. Please check back closer to the new workshop date for
-information about how to participate in the workshop.
-
-We will strive to accommodate as many as possible, but workshop space
-is limited and we are committed to ensuring a broadly diverse
-participant group, both in terms of traditional diversity measures as
-well as research domain and institutional representation.
+for participation and the rescheduled workshop, we have started anew.
```

Comment:
Should we add a phrase/sentence here that indicates that people who applied in 2020 and are still interested need to apply again? I think your sentence says that, but just to make it 100% clear?

Reply:
Yes. Good idea. Added a sentence.

Reply:
what if we add after "anew", "please register again if you are still interested"?